lib/config.js

```diff
@@ -5,10 +5,24 @@
  */
 'use strict';
 
-const {config} = require('bedrock');
-
+const {config, util} = require('bedrock');
+const cc = util.config.main.computer();
 config['session-mongodb'] = {
   collection: 'session',
-  // timeout is in seconds (time for session to live on the server)
-  ttl: 60 * 30
 };
+
+// MongoStore ttl defaults to thirtyMinutes expressed in seconds.
+const thirtyMinutes = 60 * 30;
+
+// timeout is in seconds (time for session to live on the server)
+cc('session-mongodb.ttl', () => {
+  if(config.express && config.express.session) {
+    // express session ttl is in milliseconds
+    const {ttl = thirtyMinutes * 1000} = config.express.session;
+    // round up to give the user the benefit of the doubt
+    // with time outs
+    return Math.ceil(ttl / 1000);
```

Comment:
I think we should round the other way -- but in reality, this will probably never actually even be a thing.

```suggestion
    // express session ttl is in milliseconds
    const {ttl = thirtyMinutes * 1000} = config.express.session;
    // round down to ensure session doesn't live beyond configured value
    return Math.floor(ttl / 1000);
```